sdk/include/reactos/shellutils.h

```diff
@@ -293,6 +293,17 @@ HRESULT inline ShellDebugObjectCreator(REFIID riid, R ** ppv)
     return S_OK;
 }
 
+template<class T>
+HRESULT inline ShellObjectCreator(CComPtr<T> &objref)
+{
+    _CComObject<T> *pobj;
+    HRESULT hResult = _CComObject<T>::CreateInstance(&pobj);
+    objref = pobj; // AddRef()
```

Comment:
Why commented?

Reply:
It just means AddRef gets called by the = operator. I'll clarify the comment.